dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -881,6 +881,7 @@ contenttypes.field.variables.value_no_rows.label=No Records Found
 contenttypes.field.properties.relationship.new.label=New
 contenttypes.field.properties.relationship.existing.label=Existing
 contenttypes.field.properties.relationship.existing.placeholder=Select Relationship
+contenttypes.field.properties.relationship.required.error=Automaticly was set required to false for the relationship {0} because Only one side of the relationship can be required 
```

Comment:
there is a typo in the message, I suggest:  "Required property was automatically set to false for the relationship {0} because only one side of the relationship can be required "